Open-Rival/include/UnitAnimationLookup.h

```diff
@@ -0,0 +1,179 @@
+#ifndef UNIT_ANIMATION_LOOKUP_H
+#define UNIT_ANIMATION_LOOKUP_H
+
+#include <map>
+
+namespace Rival {
+
+    enum class UnitType : int {
+
+        // Human
+        Peasant,
+        Bowman,
+        LightCavalry,
+        Knight,
+        FireMaster,
+        Thief,
+        Ballista,
+        ChariotOfWar,
+        Wizard,
+        Priest,
+        SeaBarge,
+        Battleship,
+        PegasRider,
+        Zeppelin,
+
+        // Greenskin
+        Serf,
+        RockThrower,
+        HordeRider,
+        Warlord,
+        GnomeBoomer,
+        Rogue,
+        Catapult,
+        StormTrooper,
+        PriestOfDoom,
+        Necromancer,
+        LandingCraft,
+        TrollGalley,
+        Warbat,
+        Balloon,
+
+        // Elf
+        Yeoman,
+        Archer,
+        Druid,
+        Centaur,
+        DwarfMiner,
+        Scout,
+        Bombard,
+        Arquebusier,
+        Mage,
+        Enchanter,
+        Bark,
+        Warship,
+        SkyRider,
+        MagicChopper,
+
+        // Monsters
+        Devil,
+        Dragon,
+        Golem,
+        Gryphon,
+        Hydra,
+        SeaMonster,
+        Skeleton,
+        Snake
+    };
+
+    enum class Facing : int {
```

Comment:
Out of curiosity, why have you added types to the enums? That was actually one of the things I wasn't sure about on the TODO list, I had planned to do some research about it.